applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -1312,6 +1313,34 @@ void SmallStrainUPwDiffOrderElement::CalculateAll(MatrixType&        rLeftHandSi
     KRATOS_CATCH("")
 }
 
+std::vector<double> SmallStrainUPwDiffOrderElement::CalculateDerivativesOfSaturation(
+    const std::vector<double>& fluid_pressures, RetentionLaw::Parameters& RetentionParameters)
```

Comment:
To comply with the Kratos Style Guide, we need to rename the first parameter:
```suggestion
    const std::vector<double>& rFluidPressures, RetentionLaw::Parameters& RetentionParameters)
```

And as mentioned before, I would make `RetentionParameters` a local variable, to limit its scope as much as possible.

Reply:
Done